### ISSUE TITLE
Datasets in training and schedule

### ISSUE THREAD

**Issue by orram** [megvii-research/CREStereo#45]
https://github.com/megvii-research/CREStereo/blob/ad3a1613bdedd88b93247e5f002cb7c80799762d/train.py#L147
Thank you for supplying this code and training procedure!
In the paper (and the git readme), you say you train using other datasets as well ([SceneFlow], [Sintel], [Middlebury], [ETH3D], [KITTI 2012/2015], [Falling Things], [InStereo2K], [HR-VS]). 
Yet, in the train.py, you only refer to your CRES dataset. 
Can you elaborate? Are you training on other datasets before? After? 

Thank you!

**Comment by ynma-hanvo** [comment]
yeah,  the public dataset is really too hard to use.    can share the tools you used to generate the cres dataset from public dataset?
